### DISCUSSION TITLE
Generating Multiple Texture Maps in OpenMVS' TextureMesh Step

### DISCUSSION THREAD

**Discussion by monikasarakawka** [cdcseacave/openMVS#962] [Q&A]
Hello. I was wondering if it is possible to generate multiple texture maps for a single mesh in the TextureMesh step of OpenMVS? If so, could you provide guidance on how to achieve this?

**Comment by cdcseacave** [comment]
The code does not support that for now, though would be easy to add, simply a mesh instead of keeping just one image for a texture, keep a vector of images, and a vector of indices for each face pointing to the image it projects to. Pls contribute if you can.
